trino/pom.xml

```diff
@@ -353,6 +353,7 @@
         <dependency>
             <groupId>org.openjdk.jol</groupId>
             <artifactId>jol-core</artifactId>
+            <version>${jol.version}</version>
```

Comment:
Does trino moudle still need this dependency `org.openjdk.jol`?

If trino is the only module that must depend on `org.openjdk.jol`, maybe we should declare `jol.version` in the properties of `trino/pom.xml`.

Reply:
Yes, I have moved it.